README.md

```diff
@@ -15,12 +15,12 @@ Before using this plugin, you must ensure that `eslint_d` is installed on your s
 
 1. Install `eslint_d` by typing the following in a terminal:
    ```
-   npm install -g eslint_d
+   npm install -g eslint_d@>=2.3.1
```

Comment:
I don't think it should be version here and below, it will be too confusing for users, let npm just to install latest version.